webpack/base.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: ['./src/index.tsx'],
```

Comment:
For at prod bundle skal funke riktig, burde du bruke denne:

entry: {
    main: './src/index.tsx'
  }